### PR TITLE
feat: apply update UI for app

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA
image

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/administration/store/app-page-store.ts [hunk 1]
```diff
@@ -52,6 +53,17 @@ export const useAppPageStore = defineStore('app-page', {
...
+            try {
+                await SpaceConnector.clientV2.identity.app.update<AppUpdateParameters, AppModel>(params);
+            } catch (e) {
+                ErrorHandler.handleError(e);
+                throw e;
+            } finally {
+                this.modal.loading = false;
+            }
+        },
         //
```
Comment (piggggggggy) [inline apps/web/src/services/administration/store/app-page-store.ts:67]: Question: "//" typo?

File: apps/web/src/services/administration/components/AppManagementTable.vue [hunk 2]
```diff
@@ -94,15 +94,34 @@ const dropdownMenu = computed<MenuItem[]>(() => ([
 /* Component */
 const handleSelectDropdown = (name) => {
     switch (name) {
-    case APP_DROPDOWN_MODAL_TYPE.EDIT: clickFormModal(name);
+    case APP_DROPDOWN_MODAL_TYPE.EDIT: clickFormModal({
+        name,
+        title: i18n.t('IAM.APP.MODAL.EDIT_TITLE') as string,
+    });
```
Comment (piggggggggy) [inline apps/web/src/services/administration/components/AppManagementTable.vue:100]: Long time no see..

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Pull request merged**